pkg/apis/core/v1alpha1/portforward_types.go

```diff
@@ -125,14 +130,29 @@ func (in *PortForwardList) GetListMeta() *metav1.ListMeta {
 
 // PortForwardStatus defines the observed state of PortForward
 type PortForwardStatus struct {
-	// Time at which we started trying to run the Port Forward (potentially distinct
-	// from the time the Port Forward successfully connected)
-	StartedAt metav1.MicroTime `json:"startedAt,omitempty" protobuf:"bytes,1,opt,name=startedAt"`
```

Comment:
I just deleted this field since it wasn't being populated yet, but I did force the protobuf numbering to re-start at 2 just to be safe.